scripts/feed_view.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable no-unused-vars */
+/**
+ * FeedView class
+ */
+class FeedView {
+    tweetViewer = new TweetView(true);
+    _addTemplate =
+        document.getElementById('add-template');
+    feed = document.querySelector('[class="feed"]');
```

Comment:
document.querySelector('.feed');

Reply:
Btw, but if it was document.querySelector('[data-role="feed"]');
That will be cool approach, as big part of community use attributes as markers for js.